site/gatsby-site/src/graphql/checklists.js

```diff
@@ -53,6 +54,14 @@ export const UPDATE_CHECKLIST = gql`
   }
 `;
 
+export const INSERT_EMPTY_CHECKLIST = gql`
+  mutation insertChecklist {
+    insertOneChecklist {
+      _id
+    }
+  }
+`;
+
```

Comment:
I didn't find where is this mutation being used. Please delete this declaration statement if it's not used.